operator/config/crd/bases/servicebroker.metabroker.suse.com_offerings.yaml

```diff
@@ -32,15 +32,24 @@ spec:
         metadata:
           type: object
         spec:
-          description: OfferingSpec defines the desired state of Offering
+          description: OfferingSpec defines the Service Offering spec.
```

Comment:
I don't think you want `spec` there, unless you mean you're referring to an external specification (in which case you should link to it); it's kind of confusingly ambiguous here.

Reply:
Done with https://github.com/SUSE/metabroker/pull/6/commits/af94989eecc3428dc43509716746d5610a2994a8.